src/vm_interrupts.rs

```diff
@@ -87,17 +87,19 @@ pub struct VmCpuExtInterrupts {
     imsic_location: ImsicLocation,
     sw_file: SwFile,
     allowed_ids: AllowList,
+    num_guests: usize,
 }
 
 impl VmCpuExtInterrupts {
     /// Creates a new `VmCpuExtInterrupts` to track the external interrupt state of a vCPU with
     /// `imsic_location` as the location of the vCPU's virtualized IMSIC.
```

Comment:
nit: num_guests is pretty self explanatory, but would be nice to have in the doc comment .

Reply:
Done.